contracts/src/teleporter/registry/TeleporterOwnerUpgradeable.sol

```diff
@@ -22,7 +22,15 @@ abstract contract TeleporterOwnerUpgradeable is TeleporterUpgradeable, OwnableUp
         address initialOwner
     ) internal onlyInitializing {
         __TeleporterUpgradeable_init(teleporterRegistryAddress);
-        __Ownable_init();
+        __Ownable_init_unchained();
```

Comment:
IIUC, `__Ownable_init` vs `__Ownable_init_unchained` doesn't have any functional difference here. 

For my understanding, is there a reason why `__Ownable_init_unchained` should be preferred? Seems like the normal `_init` would be recommended unless there is a specific need to ensure none of the `Ownable ` parent initializers are called